src/Components/Counter.jsx

```diff
@@ -1,15 +1,43 @@
-import React from 'react';
-import Display from './Display';
+import React, {useState} from 'react';
+import  "./DisplayCss.css";
+import Display from "../Components/Display"
 import ButtonPad from './ButtonPad';
 
 
-function Counter() {
+const Counter=()=> {
+    const [count, setCount] = useState(0);
+    const [list, setList]= useState([]);
+    const handleCount = (handle)=>{
+        if(handle == 0){
+            setCount(0)
+            setList([0])
+            return 
```

Comment:
rather then doing a return, why not do an if else?  it is easier to read with a simple if / else